web/src/engine/osk/src/views/floatingOskView.ts

```diff
@@ -120,7 +110,7 @@ export default class FloatingOSKView extends OSKView {
       this.footerView = null;
     }
 
-    this.loadCookie();
+    this.loadPersistedLayout();
```

Comment:
The main reason for the name-change:  SHIFT-CTRL-F.

Changing the name means it doesn't show up in project-wide searches for remaining uses of `loadCookie` / `saveCookie`, helping to track how much of that work is done.